WordPressVIPMinimum/Sniffs/AbstractVariableRestrictionsSniff.php

```diff
@@ -18,7 +18,7 @@
  *
  * @package VIPCS\WordPressVIPMinimum
  */
-abstract class AbstractVariableRestrictionsSniff extends Sniff {
+abstract class AbstractVariableRestrictionsSniff extends VIPCSSniff {
```

Comment:
Instead of using a class with a different name, we should be able to remove the `use WordPress\Sniff;` and `use PHP_CodeSniffer\Sniffs\Sniff` import statements from each file that uses it, and keep `VIPCSSniff` as `Sniff`.